packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```diff
@@ -360,18 +360,11 @@
             "description": "The dashboard name. Must match the 'name' attribute in the dashboard file."
           },
           "condition": {
-            "type": ["null", "string"],
-            "description": "If met, the dashboard is created in the app. If not met, the dashboard is not created.",
-            "defaultSnippets": [
-              {
-                "label": "\"\"",
-                "body": "${0}"
-              },
-              {
-                "label": "${...} expression",
-                "body": "${Variables.${1:varname}}"
-              }
-            ]
+            "$ref": "#/definitions/condition",
+            "description": "If met, the dashboard is created in the app. If not met, the dashboard is not created."
+          },
+          "onFailure": {
```

Comment:
@karihotchkiss - it looks like `onFailure` should work for all of the Conditional asset types (except for image files), right?

Reply:
That's correct.  An onFailure definition for image files will be ignored (we can change if necessary, though)